### PR TITLE
Feature/airtime/wk

### PR DESCRIPTION
**Pull request by Wesly1924**
# Description
 
Added the Prepaid Pinless Airtime Credit Request Use Case: Created a detailed description of the Prepaid Pinless Airtime Credit Request use case. Included the desired outcome, preconditions, participants, workflow, and happy path for the use case. Added a sequence diagram illustrating the happy path for the Prepaid Pinless Airtime Credit Request.
 
Fixes # (issue)
 
## Type of change
 
Please delete options that are not relevant.
 

- [ x] New feature (non-breaking change which adds functionality)

 
# How Has This Been Tested?
 
It was run on github preview and vscode preview 
 
-
 
# Checklist:
 
- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ x] My changes generate no new warnings
- [ x] I have added / ran tests that prove my fix is effective or that my feature works
- [ x] New and existing unit tests pass and existing features work locally with my changes
- [ ] My changes introduce new infrastructure concerns and/or new package dependencies

### CODE REVIEW

**Review by Keabetswe-mambule — CHANGES_REQUESTED** [review]
Add methods description table for all products

Remove API keys and passwords from all postman collections and use {{ placeholders }} instead

Add the word "methods" on all methods sections for all products 

Clearly separate pines and pin(voucher) airtime use cases

File: documentation/multichoice.md [hunk 1]
```diff
@@ -0,0 +1,584 @@
...
+
+fetch("https://tps.prepaidplus.co.bw/apimanager/rest/basic/v1/dstv/advice", requestOptions)
+  .then(response => response.text())
+  .then(result => console.log(result))
+  .catch(error => console.log('error', error));
+````
+
+**Example Response**
+
+This example above makes a request for clientSaleId:12349876 which for purposes of illustration had timed out, whilst the call had resulted in a successful sale.
```
Comment (Keabetswe-mambule) [inline documentation/multichoice.md:552]: does the request above really use the above this clientSaleId:12349876? Please do the needful

**Review by Keabetswe-mambule — COMMENTED** [review]
Spacing above the first use cases is not there for electricity and airtime

File: documentation/multichoice.md (hunk 1 above)
Comment (Wesly1924) [reply documentation/multichoice.md:552]: ok let me look into it

**Pull request closed without merging**